src/TransitionGroup.js

```diff
@@ -6,16 +6,19 @@ import { assign, linkRef } from './util';
 const identity = i => i;
 
 export class TransitionGroup extends Component {
-	static defaultProps = {
```

Comment:
Buble doesn't support the property initializer syntax (Rich-Harris/buble#123) nor the `static` keyword so I had to rewrite this code